app/src/main/java/com/example/triptracker/model/repository/Response.kt

```diff
@@ -0,0 +1,9 @@
+package com.example.triptracker.model.repository
+
+sealed class Response<out T> {
+  data object Loading : Response<Nothing>()
+
+  data class Success<out T>(val data: T?) : Response<T>()
+
+  data class Failure(val e: Exception) : Response<Nothing>()
+}
```

Comment:
Even if the behavior is really straightforward, would be nice if you add small quick comments to describe :))

Reply:
I added comments thanks ! I forgot